packages/cli/lib/connectors.json

```diff
@@ -112,20 +112,19 @@
     "settings": {
       "dsn": {
         "type": "string",
-        "message": "ODBC DSN to use for the connection"
+        "message": "ODBC DSN"
       },
       "connectionString": {
         "type": "string",
-        "message": "ODBC connection string for connecting to the database (eg: DSN=*LOCAL;UID=MY_UID;PWD=MY_PWD)"
+        "message": "ODBC connection string for overriding other options (e.g.: DBQ=QGPL;NAM=1)"
       },
-      "username": {
-        "type": "string"
+      "user": {
+        "type": "string",
+        "message": "IBM i User (If not specified in DSN or connection string)"
       },
       "password": {
-        "type": "password"
-      },
-      "schema": {
-        "type": "string"
+        "type": "password",
+        "message": "IBM i User's Password (If not specified in DSN or connection string)"
       }
     },
     "package": {
```

Comment:
> The CLI is unable to rectify the connectors.json connectors property name (db2i) with the "name" property (ibmi), which then causes it to download the package ibmi, which is incorrect (should download loopback-connector-ibmi). Also doesn't ask about the various settings

Hmm, I wonder which package it downloaded before the change. Seems like the PR only changes some prompts and their messages. From my understanding, it should still download `loopback-connector-ibmi` without the change. Anyway it LGTM 👍

Reply:
It was downloading the `ibmi` package. 

https://github.com/strongloop/loopback-next/blob/600504525d5bcaf121075f3699cedc36b93292e5/packages/cli/generators/datasource/index.js#L300-L309

Here, connectors was looking for the key `ibmi`, but since the property was called `db2i`, `connector` became undefined, which made it drop through to the else statement, and then download `ibmi` package (and completely skipping all of the JSON inside the `db2i` property)

Reply:
Make sense, thanks!